src/langcheck/metrics/zh/source_based_text_quality.py

```diff
@@ -86,8 +86,11 @@ def factual_consistency(
 
     global _factual_consistency_translation_pipeline
     if _factual_consistency_translation_pipeline is None:
```

Comment:
I don't think we need to make this a global var anymore, since the model manager is maintaining it?